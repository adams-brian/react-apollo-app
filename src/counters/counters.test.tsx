// import { shallow } from 'enzyme';
// import * as React from 'react';
// // import * as sinon from 'sinon';

// import Counters from './counters';

describe("Counters", () => {

  // const list = [5,1,4,2,3];
  // const loaded = true;
  // const error = '';
  // const counters = { list, loaded, error };

  describe("component", () => {

    // // type propFunction = (index: number) => void;
    // let element: JSX.Element;
    // // let addCounter: sinon.SinonSpy;
    // // let increment: sinon.SinonSpy;
    // // let decrement: sinon.SinonSpy;
    // // let reset: sinon.SinonSpy;
    // // let remove: sinon.SinonSpy;
    
    // beforeEach(() => {
    //   // addCounter = sinon.spy();
    //   // increment = sinon.spy();
    //   // decrement = sinon.spy();
    //   // reset = sinon.spy();
    //   // remove = sinon.spy();

    //   element = <Counters
    //     counters={[...list]}
    //     // addCounter={addCounter}
    //     // increment={increment}
    //     // decrement={decrement}
    //     // reset={reset}
    //     // remove={remove}
    //   />
    // });

    it('renders as expected', () => {
      expect(true).toBe(true);
      // const component = shallow(element);
      // expect(component).toMatchSnapshot();
    });

  });

});
