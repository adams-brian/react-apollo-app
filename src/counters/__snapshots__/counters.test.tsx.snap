// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counters component renders as expected 1`] = `
<div
  className="counters-container"
>
  <h1>
    Counters
  </h1>
</div>
`;
